about.html

```diff
@@ -25,21 +25,21 @@
         <div class="menu-logo">Sarah's Portfolio</div>
         <div class="menu-icons">
           <ul>
-            <li><a href="/"><i class="fab fa-twitter-square fa-2x" aria-label="My Twitter account"></i></a></li>
-            <li><a href="/"><i class="fab fa-tumblr-square fa-2x" aria-label="My Tumblr account"></i></a></li>
-            <li><a href="/"><i class="fab fa-facebook-square fa-2x" aria-label="My Facebook account"></i></a></li>
-            <li><a href="/"><i class="fab fa-youtube-square fa-2x" aria-label="My YouTube account"></i></a></li>
+            <li><a href="https://twitter.com/notmclachlan/"><i class="fab fa-twitter-square fa-2x" aria-label="My Twitter account"></i></a></li>
+            <li><a href="https://notmclachlan.tumblr.com/"><i class="fab fa-tumblr-square fa-2x" aria-label="My Tumblr account"></i></a></li>
+            <li><a href="https://www.facebook.com/sarah.mocklin"><i class="fab fa-facebook-square fa-2x" aria-label="My Facebook account"></i></a></li>
+            <li><a href="https://www.instagram.com/notmclachlan/"><i class="fab fa-instagram fa-2x" aria-label="My Instagram account"></i></a></li>
           </ul>
         </div>
       </div>
       <div class="mobile-header">
         <div class="mobile-menu">
           <div class="mobile-nav">
-              <ul>
-              <li><a href="index.html">Home</a></li>
-              <li><a href="about.html">About Me</a></li>
-              <li><a href="contact.html">Contact</a></li>
-            </ul>
+              <p style="padding-top: 15px;"><ul>
```

Comment:
Try to put inline styles like this in the css file and give the paragraph a class name associated with the styling.